libmdnsd/CMakeLists.txt

```diff
@@ -72,7 +75,7 @@ else()
     add_library(libmdnsd SHARED ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
```

Comment:
On Linux, this creates a `liblibmdnsd.so` file.
You can remove the `lib` prefix,
unless this is a breaking change?

Reply:
You have the `add_library` twice here.
You can avoid it by doing:
```
option(BUILD_SHARED_LIBS "Build libmdnsd as a shared library" ON)
add_library(libmdnsd ${LIBRARY_FILES} ${PUBLIC_INCLUDES})
```